pkg/cmd/cli/install/install.go

```diff
@@ -195,19 +195,22 @@ Use '--wait' to wait for the Velero Deployment to be ready before proceeding.
 Use '-o yaml' or '-o json'  with '--dry-run' to output all generated resources as text instead of sending the resources to the server.
 This is useful as a starting point for more customized installations.
 		`,
-		Example: `	# velero install --bucket mybucket --provider gcp --secret-file ./gcp-service-account.json
+		Example: `	# velero install --provider gcp --plugins velero/velero-plugin-for-gcp:v1.0.0 --bucket mybucket --secret-file ./gcp-service-account.json
```

Comment:
Thanks for catching this! It really should have been added when the `plugins` flag was.